### ISSUE TITLE
Issue running this new version

### ISSUE THREAD

**Issue by palevelmode** [rabobank-cdc/DeTTECT#27]
Hi DeTTECT

I would like to seek assistance, I am having error change I change the path. I am using windows 10 and all version of application is the latest one.

![image](https://user-images.githubusercontent.com/6778643/78239951-ee876a80-7510-11ea-88e2-dc0d85e87cdc.png)

**Comment by rubinatorz** [comment]
Hi @palevelmode,

Do I understand it correctly that you were using a previous version of DeTT&CT before? Then maybe cleaning up the cache folder within your DeTT&CT folder could be of help.

When that doesn't solve your problem, can you please tell us what you are trying to do? Please mention the steps in the interactive menu or share the command line you used.

**Comment by palevelmode** [comment]
Hi,

I  install the another instance of this latest version in venv under windows 10 ( I have an old version of this that I installed sometime early 2019 but when  I check it last night I had different error.)

I only trying to change the path of yaml files using "python dettect.py -i"

**Comment by rubinatorz** [comment]
Hi @palevelmode 

It took me some time but I found the problem. This week a new version of the taxii2-client python library was released: version 2.0.0. This library uses TAXII 2.1 classes by default, but this version is not supported by MITRE's TAXII server. So if you look at your installed python packages you will see that you have version 2.0.0 of the taxii2-client library. If you uninstall this version with "pip uninstall taxii2-client" and then install version 1.0.1 with "pip install taxii2-client==1.0.1" your problem will be solved.

I'm in contact with Roberto Rodriguez of the attackcti library to tackle this.
